docs/_sources/usage.rst.txt

```diff
@@ -64,6 +64,42 @@ First, setup Homebrew [#homebrew]_ and afterwards install the RTRlib package:
 
 .. [#homebrew]  Homebrew -- http://brew.sh
 
+
+Gentoo
+------
+
+The kind guys from the FRRouting project maintain a gentoo overlay [#overlay]_ that contains an ebuild for the RTRlib.
+First, setup layman [#layman]_, then install rtrlib with the following commands:
+
+.. code-block:: bash
+
+    # If this doe not work try layman -f
+    layman -a frr-gentoo
+    emerge rtrlib
+
+.. rubric:: Footnotes
+
+.. [#overlay] https://github.com/FRRouting/gentoo-overlay
+.. [#layman] https://wiki.gentoo.org/wiki/Layman
+
+
+Debian
+------
+
```

Comment:
a link to https://packages.debian.org/search?searchon=names&keywords=librtr would be handy

Reply:
Why?

Reply:
To easily be able to checkout in which apt-repositories the rtrlib is available, which rtrlib debian packages exist and in which versions

Reply:
this updates the compiled version of the handbook, if you want that link in please provide a PR that changes/adds that in the source file (and recompile again).